pkgs/servers/web-apps/wordpress/default.nix

```
@@ -18,6 +18,16 @@ stdenv.mkDerivation rec {
     inherit (nixosTests) wordpress;
   };
 
+  passthru.updateScript = writeScript "update.sh" ''
+    #!/usr/bin/env nix-shell
+    #!nix-shell -i bash -p common-updater-scripts jq wp4nix
```

```suggestion
    #!nix-shell -i bash -p common-updater-scripts jq
```

Unclear to me where wp4nix would be use in this script.